space/internal/components/migration/runner.go

```diff
@@ -26,7 +25,7 @@ const (
 var log = logger.NewNamed(CName)
 
 type Migration interface {
-	Run(context.Context, logger.CtxLogger, dependencies.QueryableStore, dependencies.QueryableStore, dependencies.SpaceWithCtx) (toMigrate, migrated int, err error)
+	Run(context.Context, logger.CtxLogger, objectstore.ObjectStore, dependencies.SpaceWithCtx) (toMigrate, migrated int, err error)
```

Comment:
We need to introduce new interface in `space/internal/components/dependencies`.
It could be called SpaceIndexGetter and implement method
`SpaceIndex(spaceId string) QuaryableStore`

Reply:
This dependency is needed to guarantee migration will not use data changing methods of ObjectStore